ios/FluentUI.Demo/FluentUI.Demo/Demos/ContactCollectionViewDemoController.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+
+class ContactCollectionViewDemoController: DemoController {
+    private let contactCollectionView = ContactCollectionView()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        contactCollectionView.translatesAutoresizingMaskIntoConstraints = false
+        contactCollectionView.contactList = samplePersonas
+
+        view.addSubview(contactCollectionView)
+
+        contactCollectionView.leadingAnchor.constraint(equalTo: view.leadingAnchor).isActive = true
```

Comment:
Activate all constraints at the same time like this: NSLayoutConstraints.activateConstraints([
   contactCollectionView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
   contactCollectionView.trailingAnchor.constraint(equalTo: view.trailingAnchor)
])